src/main/docker/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  gerrit:
+    image: plugin-test:latest
+    ports:
+      - "8080:8080"
+      - "29418:29418"
+    networks:
+      - host-net
```

Comment:
Don't we want a rabbitmq instance here as well, if user/developer want to try out Gerrit with Eiffel plugin?
If you think we should complement this with services from Easy2Use Eiffel bundle and this docker-compose,, then you should default network name to same network name as we have in Easy2Use Eiffel bundle,, or what Docker enviornment we would like to start it together with.

Reply:
The docker compose file may need changes in the future.,